### PR TITLE
[Postgres] fix lose precision in PostgresParallelSource

### PR DESCRIPTION
**Pull request by wuzhenhua01**
fix https://github.com/ververica/flink-cdc-connectors/issues/2391

### CODE REVIEW

File: flink-connector-postgres-cdc/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java [hunk 1]
```diff
@@ -53,6 +52,8 @@
 /**
  * {@link JdbcConnection} connection extension used for connecting to Postgres instances.
  *
+ * <p>Line 755 : cherry pick from DBZ-5470
```
Comment (loserwang1024) [inline flink-connector-postgres-cdc/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java:55]: Add: Remove it when debezium version is upgraded above 2.0.0.Final.

**Comment by loserwang1024** [comment]
Great Catch, Please rebase this PR.

File: flink-connector-postgres-cdc/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java (hunk 1 above)
Comment (wuzhenhua01) [reply flink-connector-postgres-cdc/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java]: done

**Comment by loserwang1024** [comment]
LGTM. CC, @leonardBang , @ruanhang1993

**Comment by yuxiqian** [comment]
Thanks @wuzhenhua01 for the great work! Before this PR could be merged, could you please rebase it with latest `master` branch?

cc @loserwang1024

**Comment by loserwang1024** [comment]
@wuzhenhua01 Please rebase this pr to master.

**Comment by wuzhenhua01** [comment]
done

**Pull request closed without merging**